### PR TITLE
Introduce loading state to encrypt and decrypt account

### LINKED ISSUE #5107: Introduce loading state for account encryption and decryption
### Description

With the `memorySize` changes [introduced]( https://github.com/LiskHQ/lisk-sdk/commit/377de0ce8d55ada99069eaf7b305d00036b42519) for account encryption and decryption, the UI appears to hang as we do not have loading state for the waiting interval.

### Acceptance Criteria

- Add loading state to as [proposed](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?type=design&node-id=5970%3A330635&mode=design&t=hL05V0aA9QwpUTgy-1)

Impacted area
- Encrypt account (Add account
- Decrypt account (All password prompts, Backup account)

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #5107

### How was it solved?

- Introduced the SVGr webpack loader
- Created a new prop `isLoading` for buttons
- Updated unit tests

### How was it tested?

- Navigate to send token modal
- Provide necessary details on the send form
- Move through all steps until you are at the password collection step
- Provide your password. At this stage account decryption is done thought the loader would be to fast to see.

### CODE REVIEW

File: src/theme/buttons/index.js [hunk 1]
```diff
@@ -5,14 +5,25 @@ import secondaryButtonTheme from './css/secondaryButton.css';
 import styles from './css/size.css';
 import tertiaryButtonTheme from './css/tertiaryButton.css';
 import warningButtonTheme from './css/warningButton.css';
+import baseStyles from './css/base.css';
 import outlineButtonTheme from './css/outlineButton.css';
+import { ReactComponent as CircularLoader } from '../../../setup/react/assets/images/icons/loading-rounded.svg';
```
Comment (oskarleonard) [inline src/theme/buttons/index.js:10]: alias

File: src/theme/buttons/index.test.js [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
 import { Button } from './index';
 
 describe('Button wrapper', () => {
   it('Creates only one instance of the button', () => {
-    const wrappedButton = mount(<Button />, {});
-    expect(wrappedButton.find('Button').length).to.equal(1);
+    const button = render(<Button>test</Button>);
+    expect(button.getAllByText('test').length).toEqual(1);
+  });
+  it('renders loading state', () => {
```
Comment (ManuGowda) [inline src/theme/buttons/index.test.js]:
```suggestion
  });

  it('renders loading state', () => {
```

**Review by oskarleonard — APPROVED** [review]

**Review by oskarleonard — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]

**Review by ManuGowda — DISMISSED** [review]

**Review by oskarleonard — DISMISSED** [review]

File: src/modules/account/components/AddAccountBySecretRecovery/AddAccountBySecretRecovery.test.js [hunk 3]
```diff
@@ -67,17 +68,7 @@ describe('Add account by secret recovery phrase flow', () => {
...
-        screen.getByText(
-          'You can now download your encrypted secret recovery phrase and use it to add your account on other devices.'
-        )
-      ).toBeTruthy();
-      expect(screen.getByText('Download')).toBeTruthy();
-
-      fireEvent.click(screen.getByText('Continue to wallet'));
-
-      expect(props.history.push).toBeCalled();
+      expect(mockOnMessage).toBeCalled();
```
Comment (ManuGowda) [inline src/modules/account/components/AddAccountBySecretRecovery/AddAccountBySecretRecovery.test.js]: You could have checked why message is passed? in this case the plaintext and password to be called?

**Review by ManuGowda — COMMENTED** [review]
The worker is not terminated? https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#terminating_a_worker

File: src/modules/auth/components/SetPasswordForm/SetPasswordForm.js [hunk 4]
```diff
@@ -75,18 +80,27 @@ function SetPasswordForm({ prevStep, onSubmit, recoveryPhrase, customDerivationP
       });
       return null;
     }
-    const { error, result } = await encryptAccount({
-      recoveryPhrase: recoveryPhrase.value,
-      password: values.password,
-      name: values.accountName,
+
+    const encryptWorker = new Worker(new URL('./encryptAccount.worker.js', import.meta.url));
```
Comment (ManuGowda) [inline src/modules/auth/components/SetPasswordForm/SetPasswordForm.js]:
```suggestion
    const encryptAccountWorker = new Worker(new URL('./encryptAccount.worker.js', import.meta.url));
```

File: src/modules/auth/components/SetPasswordForm/SetPasswordForm.js [hunk 5]
```diff
@@ -75,18 +80,27 @@ function SetPasswordForm({ prevStep, onSubmit, recoveryPhrase, customDerivationP
...
+
+    encryptWorker.postMessage({
+      customDerivationPath,
+      recoveryPhrase,
+      enableAccessToLegacyAccounts,
+      ...values,
     });
 
-    if (error) {
```
Comment (ManuGowda) [inline src/modules/auth/components/SetPasswordForm/SetPasswordForm.js:84]: error handling is missing from worker? `encryptWorker.onerror`?

**Review by oskarleonard — APPROVED** [review]

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**